### DISCUSSION TITLE
Support for simple pagination.

### DISCUSSION THREAD

**Discussion by ssarljames** [livewire/livewire#1304] [4. General]
Hi there, 

I just want to ask if there is an existing way to implement `simplePaginate()` on Livewire Components

The `paginate()` works fine with `Livewire\WithPagination` trait but not working on `simplePaginate()` because when clicking the next/prev button, it goes to a separate link, not asynchronously.

Thanks.

**Comment by henrywhitaker3** [comment]
You ever work this out?
